### ISSUE TITLE
incorrect warning about return type of built-in CalledProcessError.stdout()

### ISSUE THREAD

**Issue by vitaly-zdanevich** [PyCQA/pylint#1235]
### Steps to reproduce
try this code against pylint:
```
import subprocess

try:
    subprocess.check_output('git tag -a 1',
                             shell=True, universal_newlines=True,
                             stderr=subprocess.STDOUT)
except subprocess.CalledProcessError as e:
    if e.stdout.endswith('already exists\n'):
        print('tag already exists')
```

### Current behavior
`E:  6, 7: Method 'stdout' has no 'endswith' member (no-member)`

### Expected behavior
no warning - this is correct - buit-in library's method stdout() return `str`

### pylint --version output
```
No config file found, using default configuration
pylint 1.6.4,
astroid 1.4.8
Python 3.5.2 (default, Oct 11 2016, 04:59:56)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```

**Comment by PCManticore** [comment]
This is a duplicate of https://github.com/PyCQA/pylint/issues/870.
